### DISCUSSION TITLE
Broaden the administration?

### DISCUSSION THREAD

**Discussion by DrMichael** [teslamate-org/teslamate#3384] [General]
Hi @adriankumpf 

given that there are 53 PR's pending, wouldn't it make sense to allow more people to confirm those? Probably some are easy to confirm, while others require Elixir or Grafana knowhow.

So, I think a team could help here.

Michael

PS: We all appreciate your work and thank you for that...

**Comment by parkr** [comment]
+1 here. It can feel like a big lift to find help, but if there's anyone in the PRs already leaving good reviews, you could tap that person for help. Maintainer documentation is really helpful too. What's the best way to get a dev environment setup, what to look for in PRs, etc.

Another option is to make things more modular, so that you can focus on the Elixir code and others could work on the Grafana side of things. Using CODEOWNERS would allow you to set yourself as a required reviewer of elixir code so that no changes are made without your approval.

**Comment by djpalmis** [comment]
Shall we move to a new fork? Which one?

**Reply by parkr** [reply]
I'd highly recommend moving to an organization like GitHub.com/teslamatefork and adding at least 3 total contributors so the lottery factor is not 1 like it is presently.

**Comment by NirKli** [comment]
> Hi @adriankumpf
> 
> given that there are 53 PR's pending, wouldn't it make sense to allow more people to confirm those? Probably some are easy to confirm, while others require Elixir or Grafana knowhow.
> 
> So, I think a team could help here.
> 
> Michael
> 
> PS: We all appreciate your work and thank you for that...

+1
Thank you so much for your hard work, @adriankumpf

**Reply by jlestel** [reply]
given the lack of response for several months from adrian, I created the fork: https://github.com/MyTeslaMate/teslamate

I retrieved all the PRs in this repo and the CI works: https://github.com/MyTeslaMate/teslamate/pulls

I'm looking to build a team for this repo so the community can get the attention it deserves. I need at least 2 admins with me.

If there are people who want to help me bring the project to life, they are welcome!

**Reply by Dulanic** [reply]
I'm happy to help out and have done a few PR's in the past. However. my elixir knowledge is limited so I can't help as much there on the elixir side.

**Comment by adriankumpf** [comment]
Hey @DrMichael, I've sent you an invitation to become a collaborator. I'm willing to invite more maintainers to this repository, but I also support forks, since I don't plan to actively develop TeslaMate any further after releasing an update to address #3393.

**Reply by cwanja** [reply]
> Hey @DrMichael, I've sent you an invitation to become a collaborator. I'm willing to invite more maintainers to this repository, but I also support forks, since I don't plan to actively develop TeslaMate any further after releasing an update to address #3393.

That's sad to hear, but also understandable. Thanks for the amazing application over the years.

**Reply by MrNickIE** [reply]
I agree, but thank you so much for this great tool.   Your service is very much appreciated @adriankumpf !

**Reply by parkr** [reply]
Thank you so much for your hard work, @adriankumpf! Teslamate is a real gift to the Tesla community. Unless you have objections, continuing Teslamate in this repo feels like the best option IMO since there's infra (like docs.teslamate.org) that is already built around this project. 

Would you be willing to mentor a maintainer or two on an adhoc email basis so things like how to cut a release and things to look out for in an elixir and/or grafana PR could be more seamlessly handled by the new guard? Thanks again for all your hard work!

**Reply by parkr** [reply]
Also, it would make sense to me to move this to a @teslamate org so @DrMichael can add new collaborators and otherwise manage the project without needing your input in the future!

**Reply by markusdd** [reply]
Thanks Adrian for stepping in here and all you've done for this project so far, happy to hear we will have broader administration to move this further!

**Reply by adriankumpf** [reply]
@parkr, yeah, we definitely want to have a dedicated organization. TeslaMate now lives at @teslamate-org.

**Reply by DrMichael** [reply]
Thanks @adriankumpf for this and for all your work in the past! Now we need to organise ourselves... :-)

**Reply by jheredianet** [reply]
Thank you so much for your hard work, @adriankumpf! Long live to Teslamate :-)

**Comment by DrMichael** [comment]
How are we moving forward from here?

- Can I or shall I add more people to teslamate-org?
- Can I allow more people to confirm pull requests? Or edit the code? For that I am thinking of @cwanja @Dulanic. Perhaps @jlestel and @JakobLichterfeld might want to join also? More suggestions?

**Reply by JakobLichterfeld** [reply]
I am happy to help.

Perhaps @jsight want to help as well?

**Reply by NirKli** [reply]
Excited to see the opportunity to join the TeslaMate project! I’m keen to contribute and bring my skills on board

**Reply by jlestel** [reply]
I'm in :)

**Reply by jheredianet** [reply]
@DrMichael  I'm happy to help as well.

**Reply by jsight** [reply]
I'm happy to help out with occasional MR review or fixes. :)

**Comment by DrMichael** [comment]
To be continued in https://github.com/teslamate-org/teslamate/discussions/3416
